cmd/ingester/app/consumer/offset/concurrent_list.go

```diff
@@ -15,25 +15,33 @@
 package offset
 
 import (
+	"errors"
 	"sync"
 )
 
+var errExceededSize = errors.New("list full")
+
 // ConcurrentList is a list that maintains kafka offsets with thread-safe Insert and setToHighestContiguous operations
 type ConcurrentList struct {
 	offsets []int64
+	size    int
```

Comment:
please document it

Reply:
How/What additional information to include? This var stores the maximum size of the list. Would it be better to rename to `capacity`?

Reply:
it wasn't clear if it's current size, or max size

Reply:
updated